thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/investigation-link/template.hbs

```diff
@@ -0,0 +1,4 @@
+{{#link-to 'rootcause' (query-params anomalyId=record.id)
+  class="te-anomaly-table__link"}}
```

Comment:
we're already have styling for `thirdeye-link`, if this is slightly different, how about adding a modifier like `thirdeye-link--large`

Reply:
@justYves This is temporary. I'm planning on trying to incorporate Steve's `te-anomaly-table` styles, which contain styles for this.